nion/swift/model/DisplayItem.py

```diff
@@ -1998,11 +1999,17 @@ def append_display_data_channel(self, display_data_channel: DisplayDataChannel,
             self.__add_display_layer_auto(display_layer, display_data_channel)
 
     def __get_unique_display_layer_color(self) -> str:
-        existing_colors = {display_layer.fill_color for display_layer in self.display_layers}
-        for color in ('#1E90FF', "#F00", "#0F0", "#00F", "#FF0", "#0FF", "#F0F", "#888", "#800", "#080", "#008", "#CCC", "#880", "#088", "#808", "#964B00"):
-            if not color in existing_colors:
-                return color
-        return '#1E90FF'
+        existing_colors = [display_layer.fill_color for display_layer in self.display_layers]
+        unused_colors = list({*Color.svg_color_map.values()})
```

Comment:
I think this code can be simplified... just get the existing colors and then iterate through the color list looking for the first one that doesn't match anything in the existing colors and use it.

Reply:
I revised the code to be a bit more simple, but I think any more simplification would require the ability to sort and or directly have casting to  9 character hex colors